src/envs/blocks.py

```diff
@@ -312,8 +324,14 @@ def _sample_state_from_piles(self, piles: List[List[Object]],
             z = self.table_height + self.block_size * (0.5 + pile_j)
             # [pose_x, pose_y, pose_z, held]
             data[block] = np.array([x, y, z, 0.0])
-        # [fingers]
-        data[self._robot] = np.array([1.0], dtype=np.float32)
+        # [pose_x, pose_y, pose_z, fingers]
+        # Note: the robot poses are not used in this environment, but they
```

Comment:
when you say "not used" do you mean they are always constant? or they're updated but not used for any logic within the simulator?

Reply:
they're constant within this class

Reply:
update comment to reflect that